### ISSUE TITLE
`default_broadcastable: true` raises error

### ISSUE THREAD

**Issue by rgraff** [rmosolgo/graphql-ruby#3028]
**Describe the bug**

When using subscriptions with `default_broadcastable: true` the fingerprint query attempts to call undefined methods size on ActionController::Parameters. 

```ruby
      use GraphQL::Pro::AblySubscriptions, redis: $graphql_subscriptions_redis, ably: $ably, broadcast: true, default_broadcastable: true
```

This error is not raised withe `default_broadcastable: false`

Trace:

```
undefined method `size' for #<ActionController::Parameters:0x00005567e61fd8f0>
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/query.rb:291:in `variables_fingerprint'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/query.rb:281:in `fingerprint'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/subscriptions/event.rb:63:in `fingerprint'
/usr/local/bundle/gems/graphql-pro-1.14.0/lib/graphql/pro/subscriptions/redis_storage.rb:34:in `map'
/usr/local/bundle/gems/graphql-pro-1.14.0/lib/graphql/pro/subscriptions/redis_storage.rb:34:in `write_subscription'
/usr/local/bundle/gems/graphql-pro-1.14.0/lib/graphql/pro/subscriptions.rb:82:in `write_subscription'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/subscriptions/instrumentation.rb:34:in `after_query'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/execution/instrumentation.rb:82:in `public_send'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/execution/instrumentation.rb:82:in `block in call_after_hooks'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/execution/instrumentation.rb:80:in `reverse_each'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/execution/instrumentation.rb:80:in `call_after_hooks'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/execution/instrumentation.rb:74:in `call_hooks'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/execution/instrumentation.rb:44:in `each_query_call_hooks'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/execution/instrumentation.rb:27:in `block in apply_instrumenters'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/execution/instrumentation.rb:72:in `call_hooks'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/execution/instrumentation.rb:26:in `apply_instrumenters'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/execution/multiplex.rb:174:in `instrument_and_analyze'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/execution/multiplex.rb:60:in `block in run_queries'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/tracing.rb:66:in `trace'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/execution/multiplex.rb:58:in `run_queries'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/execution/multiplex.rb:48:in `run_all'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/schema.rb:1642:in `multiplex'
/usr/local/bundle/gems/graphql-1.11.1/lib/graphql/schema.rb:1613:in `execute'
/stellar/app/controllers/graphql_controller.rb:15:in `execute'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_controller/metal/basic_implicit_render.rb:6:in `send_action'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/abstract_controller/base.rb:195:in `process_action'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_controller/metal/rendering.rb:30:in `process_action'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/abstract_controller/callbacks.rb:42:in `block in process_action'
/usr/local/bundle/gems/activesupport-6.0.3.2/lib/active_support/callbacks.rb:135:in `run_callbacks'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/abstract_controller/callbacks.rb:41:in `process_action'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_controller/metal/rescue.rb:22:in `process_action'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_controller/metal/instrumentation.rb:33:in `block in process_action'
/usr/local/bundle/gems/activesupport-6.0.3.2/lib/active_support/notifications.rb:180:in `block in instrument'
/usr/local/bundle/gems/activesupport-6.0.3.2/lib/active_support/notifications/instrumenter.rb:24:in `instrument'
/usr/local/bundle/gems/activesupport-6.0.3.2/lib/active_support/notifications.rb:180:in `instrument'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_controller/metal/instrumentation.rb:32:in `process_action'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_controller/metal/params_wrapper.rb:245:in `process_action'
/usr/local/bundle/gems/activerecord-6.0.3.2/lib/active_record/railties/controller_runtime.rb:27:in `process_action'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/abstract_controller/base.rb:136:in `process'
/usr/local/bundle/gems/actionview-6.0.3.2/lib/action_view/rendering.rb:39:in `process'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_controller/metal.rb:190:in `dispatch'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_controller/metal.rb:254:in `dispatch'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/routing/route_set.rb:50:in `dispatch'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/routing/route_set.rb:33:in `serve'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/journey/router.rb:49:in `block in serve'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/journey/router.rb:32:in `each'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/journey/router.rb:32:in `serve'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/routing/route_set.rb:834:in `call'
/usr/local/bundle/gems/rack-2.2.3/lib/rack/tempfile_reaper.rb:15:in `call'
/usr/local/bundle/gems/rack-2.2.3/lib/rack/etag.rb:27:in `call'
/usr/local/bundle/gems/rack-2.2.3/lib/rack/conditional_get.rb:40:in `call'
/usr/local/bundle/gems/rack-2.2.3/lib/rack/head.rb:12:in `call'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/http/content_security_policy.rb:18:in `call'
/usr/local/bundle/gems/rack-2.2.3/lib/rack/session/abstract/id.rb:266:in `context'
/usr/local/bundle/gems/rack-2.2.3/lib/rack/session/abstract/id.rb:260:in `call'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/middleware/cookies.rb:648:in `call'
/usr/local/bundle/gems/activerecord-6.0.3.2/lib/active_record/migration.rb:567:in `call'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
/usr/local/bundle/gems/activesupport-6.0.3.2/lib/active_support/callbacks.rb:101:in `run_callbacks'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/middleware/callbacks.rb:26:in `call'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/middleware/executor.rb:14:in `call'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/middleware/actionable_exceptions.rb:17:in `call'
/usr/local/bundle/gems/appsignal-2.10.9/lib/appsignal/rack/rails_instrumentation.rb:19:in `call'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
/usr/local/bundle/gems/web-console-4.0.3/lib/web_console/middleware.rb:132:in `call_app'
/usr/local/bundle/gems/web-console-4.0.3/lib/web_console/middleware.rb:28:in `block in call'
/usr/local/bundle/gems/web-console-4.0.3/lib/web_console/middleware.rb:17:in `catch'
/usr/local/bundle/gems/web-console-4.0.3/lib/web_console/middleware.rb:17:in `call'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
/usr/local/bundle/gems/railties-6.0.3.2/lib/rails/rack/logger.rb:37:in `call_app'
/usr/local/bundle/gems/railties-6.0.3.2/lib/rails/rack/logger.rb:26:in `block in call'
/usr/local/bundle/gems/activesupport-6.0.3.2/lib/active_support/tagged_logging.rb:80:in `block in tagged'
/usr/local/bundle/gems/activesupport-6.0.3.2/lib/active_support/tagged_logging.rb:28:in `tagged'
/usr/local/bundle/gems/activesupport-6.0.3.2/lib/active_support/tagged_logging.rb:80:in `tagged'
/usr/local/bundle/gems/railties-6.0.3.2/lib/rails/rack/logger.rb:26:in `call'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
/usr/local/bundle/gems/request_store-1.5.0/lib/request_store/middleware.rb:19:in `call'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/middleware/request_id.rb:27:in `call'
/usr/local/bundle/gems/rack-2.2.3/lib/rack/method_override.rb:24:in `call'
/usr/local/bundle/gems/rack-2.2.3/lib/rack/runtime.rb:22:in `call'
/usr/local/bundle/gems/activesupport-6.0.3.2/lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/middleware/executor.rb:14:in `call'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/middleware/static.rb:126:in `call'
/usr/local/bundle/gems/rack-2.2.3/lib/rack/sendfile.rb:110:in `call'
/usr/local/bundle/gems/actionpack-6.0.3.2/lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
/usr/local/bundle/gems/rack-cors-1.1.1/lib/rack/cors.rb:100:in `call'
/usr/local/bundle/gems/railties-6.0.3.2/lib/rails/engine.rb:527:in `call'
/usr/local/bundle/gems/puma-4.3.5/lib/puma/configuration.rb:228:in `call'
/usr/local/bundle/gems/puma-4.3.5/lib/puma/server.rb:713:in `handle_request'
/usr/local/bundle/gems/puma-4.3.5/lib/puma/server.rb:472:in `process_client'
/usr/local/bundle/gems/puma-4.3.5/lib/puma/server.rb:328:in `block in run'
/usr/local/bundle/gems/puma-4.3.5/lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Completed 500 Internal Server Error in 24ms (Views: 1.1ms | ActiveRecord: 1.0ms | Allocations: 11676)
```

Versions:

Rails 6.0.3.2
graphql-1.11.1
graphql-pro-1.14.0

**Comment by rmosolgo** [comment]
Ick, sorry about the hangup! I ran into this issue when testing locally but apparently I hacked around it instead of upstreaming a fix. Thanks for you attached PR, I think that's :+1:

**Comment by rmosolgo** [comment]
Fixed by #3029
